src/test_s2c_srv.c

```diff
@@ -382,6 +382,9 @@ int test_s2c(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
           log_println(0, "S2C test error: can't create pipe.");
         } else {
           if ((s2c_childpid = fork()) == 0) {
+            // Don't capture more than 12 seconds of packet traces:
+            //   10 second test + 2 seconds of slop
+            alarm(12);
```

Comment:
Slop is arbitrary, the c2s test has a random 2 second sleep() in it to fix a race condition, and 10 seconds is the length of the test. I thought the comments explained it well, but I can certainly make named constants if you want. I was trying to avoid touching a lot of files.